### ISSUE TITLE
massive update of issues

### ISSUE THREAD

**Issue by luigidellaquila** [orientechnologies/prjhub#62]
especially for the milestone

**Comment by santo-it** [comment]
Yeah, important. Also to set the "Area". It seems that when issues are being opened in github and then imported here, the Area is not set. So for instance, if a studio issue is opened, when the issue is imported from the studio repo, the studio area can be set automatically as we know that is studio for sure. I'll open a new issue for this. But at least having a way to do bulk updates of issue is useful. Thanks
